contrib/completions/bash/oadm

```diff
@@ -2416,7 +2416,6 @@ _oadm_taint()
     must_have_one_noun+=("appliedclusterresourcequota")
     must_have_one_noun+=("build")
     must_have_one_noun+=("buildconfig")
-    must_have_one_noun+=("cluster")
```

Comment:
This is strange... how is it related to this PR?
@juanvallejo maybe you need a rebase?